pkg/app/piped/platformprovider/ecs/ecs.go

```diff
@@ -37,6 +38,7 @@ type ECS interface {
 	CreateService(ctx context.Context, service types.Service) (*types.Service, error)
 	UpdateService(ctx context.Context, service types.Service) (*types.Service, error)
 	RegisterTaskDefinition(ctx context.Context, taskDefinition types.TaskDefinition) (*types.TaskDefinition, error)
+	RunTask(ctx context.Context, cluster types.Cluster, awsVpcConfiguration types.AwsVpcConfiguration, taskDefinition types.TaskDefinition) (*ecs.RunTaskOutput, error)
```

Comment:
Should remove the `ecs.RunTaskOutput` from this function signature then, we can remove the `service/ecs` dependency from this file because we don't use that anyway.

Reply:
I'll fix this file to remove the `service/ecs` dependency🙇